internal/pkg/logger/http.go

```diff
@@ -250,13 +254,12 @@ func Middleware(next http.Handler) http.Handler {
 		next.ServeHTTP(wrCounter, r)
 		httpMeta(r, e)
 
-		if zlog.Debug().Enabled() || (wrCounter.statusCode < 200 && wrCounter.statusCode >= 300) {
+		if zlog.Info().Enabled() {
 			e.Uint64(ECSHTTPRequestBodyBytes, rdCounter.Count())
 			e.Uint64(ECSHTTPResponseBodyBytes, wrCounter.Count())
 			e.Int(ECSHTTPResponseCode, wrCounter.statusCode)
 			e.Int64(ECSEventDuration, time.Since(start).Nanoseconds())
-
-			e.Msgf("%d HTTP Request", wrCounter.statusCode)
+			e.Msg("HTTP Request")
```

Comment:
I've changed the middleware layer to log an info level message for each request if info is enabled.

The previous behaviour was to log an (info level) message if Debug is enabled, or the server responded in a non 2xx status

Reply:
It depends on the number of connected agents.
IIRC agents will long poll 10m by default, so we would expect a log every 10m per agent for checkins + additional entries for each action's acks

Reply:
That sounds like potentially quite a bit. Could we rate limit or aggregate these based on return code?

Reply:
FYI, I'm fine with the rest of this PR, it's just this change that seems potentially dangerous.

Reply:
actually the previous condition is broken; it would only output the (info level) message if debug is enabled.

Reply:
i'll revert & fix the condition; then make an issue for sampling logs